src/font/text.hpp

```diff
@@ -278,6 +284,21 @@ class pango_text
 
 	pango_text& set_add_outline(bool do_add);
 
+	/**
+	 * Mark a specific portion of text for highlighting. Used for selection box.
+	 * BGColor is set in set_text(), this just marks the area to be colored.
+	 * Markup not used because the user may enter their own markup or special characters
+	 * @param start_offset        Offset where selection/highlight starts
+ 	 * @param end_offset          Offset where selection/highlight ends
+ 	 * @param color               Highlight color
```

Comment:
There's weird indentation on this and the previous line - they've got a space followed by a tab.

Reply:
Updated, see if the current version is fine.
All of the doxygen comments in this file had tab followed by space.